grpc_unary_attestation/src/server.rs

```diff
@@ -140,12 +142,23 @@ where
     ) -> anyhow::Result<tonic::Response<UnaryResponse>, tonic::Status> {
         let error_logger = self.error_logger.clone();
         let request_inner = request.into_inner();
+        let session_id: SessionId = {
+            let boxed_slice: Box<SessionId> = request_inner
+                .session_id
+                .into_boxed_slice()
+                .try_into()
+                .map_err(|error| {
+                    error_logger.log_error(&format!("Received malformed session_id: {:?}", error));
+                    tonic::Status::invalid_argument("")
```

Comment:
Could you create a bug to revisit these errors (throughout the existing code) at some point? It would be useful to return **something** to the client if any error happens, as long as it does not leak any unwanted internal state or private data.

Reply:
done,  thx